packages/react-app/src/utils/miscUtils.ts

```diff
@@ -0,0 +1,14 @@
+export default {
+	csvEncode(jObj: Record<string, unknown>): Record<string, unknown> {
+		if (jObj) {
+			const jStr = JSON.stringify(jObj, (key, value) => {
+				if (typeof value === 'string') {
+					return value.replace(/"/g, '""');
+				}
+				return value;
+			});
+			return JSON.parse(jStr);
+		}
+		return jObj;
+	},
+};
```

Comment:
This works, but again not sure if it is the best way. Need to replace '"' with '""' in strings to be sent via CSV, otherwise Excel and other receivers parse things up incorrectly.

Reply:
I did some googling for you and I couldn't see a better way to be honest